cli/cobra.go

```diff
@@ -230,7 +244,7 @@ Options:
 Management Commands:
 
 {{- range managementSubCommands . }}
-  {{rpad .Name .NamePadding }} {{ if isFirstLevelCommand .}}{{commandVendor .}} {{ end}}{{.Short}}
+  {{decoratedCommandName .}}{{.Short}}{{ if isPluginFirstLevelCommand .}} {{commandVendorAndVersion .}}{{ end}}
```

Comment:
With my suggestions above, this could probably be 

```suggestion
  {{decoratedName .}}{{.Short}}{{ if isPlugin .}} {{vendorAndVersion .}}{{ end}}
```

Taking @ijc's suggestion into account;

```suggestion
  {{.rpad decoratedName .NamePadding}}{{.Short}}{{ if isPlugin .}} {{vendorAndVersion .}}{{ end}}
```

Or (if we make `vendorAndVersion` return nothing in the non-plugin case);

```suggestion
  {{.rpad decoratedName .NamePadding}}{{.Short}} {{vendorAndVersion .}}
```

Reply:
Done